src/css/styles.css

```diff
@@ -0,0 +1,468 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+header {
+  width: 100%;
+}
+
+.logo {
+  color: #6070ff;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+}
+
+body {
```

Comment:
- [ ] I think it's better to fix the design of your mobile version. Both versions must be presented on different sizes of screen. Check the content using the responsive view on Chrome Inspector. For all widths of the screen, your portfolio must present all the content.
Current view of your portfolio during the responsive view.
<img width="725" alt="Screen Shot 2022-08-25 at 10 01 51 PM" src="https://user-images.githubusercontent.com/47673311/186794909-5dc96fd9-1643-47c5-af5b-fa9a09b46506.png">

Reply:
Good job following the advice of the previous reviewer